features/images/rhel8_images.feature

```diff
@@ -0,0 +1,115 @@
+Feature: rhel8images.feature
+
+  # @author xiuwang@redhat.com
+  # @case_id OCP-22950
+  @admin
+  Scenario: Using new-app cmd to create app with ruby rhel8 image
+    Given I have a project
+    When I run the :tag admin command with:
+      | source           | registry.redhat.io/rhel8/ruby-25:latest |
+      | dest             | ruby-25-rhel8:latest                    |
```

Comment:
What will happen if the image stream with the same name already exist ?

Reply:
Just override the existing imagestream without error.

Reply:
Then it is not acceptable.
We may have that image stream integrate someday. And then this scenario will override default/existing imagestream.

Perhaps adding qe as prefix or suffix to distinguish that or you can find a better way if you do.

Reply:
I don't think it is necessary to add any xxfix, since there is already a projected created, there is an order to recognize imagestream. the top priority is to find it under its own project; default one is under Openshift namespace.

Reply:
@lxia Thanks your advice, update as comment, please help review again, thx